src/app/tabs/tabs.component.ts

```diff
@@ -50,7 +50,7 @@ export class TabsComponent extends RouterEnter {
   }
 
   private _initialise() {
-    this.showChat = true;
+    this.showChat = false;
```

Comment:
@trtshen I think we can just leave it as it is for now. This is a kind of "feature toggle". All features should have the same logic "display if there's data, don't display if there's no data".
So by default, logically we shouldn't show chat tab. It is not something we will change later, so that I think there's no need to put it to `environment.ts` to make it customisable.